### PR TITLE
chore: added assets, placeholders and tailwind config file

### PR DESCRIPTION
**Pull request by Srish-ty**
## following changes are introduced with this pr
1. content folder to save data
2. assets folder
3. a common tailwind config file 
4. updated readme with pnpm scripts and tech used

### CODE REVIEW

**Comment by Srish-ty** [comment]
@Shurtu-gal please review

File: tailwind.config.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+      },
+
+      fontFamily: {
+        Poppins: 'Poppins'
+      },
+      rounded: {
+        '4xl': '4rem'
+      },
+      screens: {
+        'brk-1': '1306px',
```
Comment (Shurtu-gal) [inline tailwind.config.js]: Should we use brk or just change the values of md, lg xl etc.

**Review by Shurtu-gal — DISMISSED** [review]
LGTM 🚀

File: tailwind.config.js (hunk 1 above)
Comment (Srish-ty) [reply tailwind.config.js]: made changes https://github.com/dscnitrourkela/project-vanilla/pull/7/commits/3aa00851a5189952e18ca367a0f50901e84b917e

**Comment by Shurtu-gal** [comment]
@Srish-ty could you resolve the prettier check.

**Comment by Srish-ty** [comment]
> @Srish-ty could you resolve the prettier check.

@Shurtu-gal  done !

**Review by Shurtu-gal — APPROVED** [review]

**Pull request merged**